src/main.qml

```diff
@@ -1288,32 +1288,6 @@ ApplicationWindow {
         id: optionspopup
         onSaveSettingsSignal: {
             imageWriter.setSavedCustomizationSettings(settings)
-            usesavedsettingspopup.hasSavedSettings = true
-        }
-    }
-
-    UseSavedSettingsPopup {
-        id: usesavedsettingspopup
-        onYes: {
-            optionspopup.initialize()
-            optionspopup.applySettings()
-            confirmwritepopup.askForConfirmation()
-        }
-        onNo: {
-            imageWriter.setImageCustomization("", "", "", "", "")
-            confirmwritepopup.askForConfirmation()
-        }
-        onNoClearSettings: {
-            hasSavedSettings = false
-            optionspopup.clearCustomizationFields()
-            imageWriter.clearSavedCustomizationSettings()
-            confirmwritepopup.askForConfirmation()
-        }
-        onEditSettings: {
-            optionspopup.openPopup()
-        }
-        onCloseSettings: {
```

Comment:
Do we need this line?

Reply:
Should be taken care of by the changes to the onClicked behavior of the next button, though as mentioned I might need to re-think that part a little bit

Reply:
Sounds good. Your idea of changing the inheritance makes a lot of sense to me, so I'd proceed with that!